frontend/micro-ui/web/micro-ui-internals/packages/modules/Measurement/src/Module.js

```diff
@@ -23,17 +23,13 @@ const MeasurementModule = ({ stateCode, userType, tenants }) => {
     if (isLoading) {
         return <Loader />;
     }
-  if (isLoading) {
-    return <Loader />;
-  }
-
 
     return <EmployeeApp path={path} stateCode={stateCode} />;
 };
 
 const componentsToRegister = {
-  MeasurementCard,
-  MeasurementModule,
+  WorksCard: MeasurementCard,
```

Comment:
this was a temporary fix to bypass mdms change,file should not be committed into repo